### ISSUE TITLE
Adjusting the focus: less data, more R

### ISSUE THREAD

**Issue by bvreede** [UtrechtUniversity/workshop-introduction-to-R-and-data#10]
Is it worth it to move the focus to R and programming, and less on data? That would include removing slides on tidy data, and removing this element from the course description.

Pro: we already have so much material to work through, and understanding data is not the primary aim of the workshop.

Con: if not now then when! Also, our central theme is data, and understanding what makes data workable is an important part of this...

**Comment by bvreede** [comment]
Considering:
- Discussing tidy data (long v wide) only once instead of twice;
- Generating a module with questions that would work as stand-alone and can be used in the R cafe as well.
